api/src/service/domain/workflow/provisioning_start.ts

```diff
@@ -0,0 +1,27 @@
+import { VError } from "verror";
+import { Ctx } from "../../../lib/ctx";
+import * as Result from "../../../result";
+import { BusinessEvent } from "../business_event";
+import { NotAuthorized } from "../errors/not_authorized";
+import { ServiceUser } from "../organization/service_user";
+import * as ProvisioningStarted from "./provisioning_started";
+
+export async function startProvisioning(
+  ctx: Ctx,
+  issuer: ServiceUser,
+): Promise<Result.Type<{ newEvents: BusinessEvent[] }>> {
```

Comment:
why the type BusinessEvent [] is wrapped in an object? Can't it be returned directly?

Reply:
done